lib/core/src/lib/form/components/widgets/core/form-field-validator.ts

```diff
@@ -192,10 +193,10 @@ export class DateTimeFieldValidator implements FormFieldValidator {
 
     validate(field: FormFieldModel): boolean {
         if (this.isSupported(field) && field.value && field.isVisible) {
-            if (DateFieldValidator.isValidDate(field.value, field.dateDisplayFormat)) {
+            if (DateTimeFieldValidator.isValidDate(field.value, DateFnsUtils.convertMomentToDateFnsFormat(field.dateDisplayFormat))) {
```

Comment:
lot of code publication with this line.... the DateTimeFieldValidator should encapsulate the conversion . use the original code and update the validator instead: https://github.com/Alfresco/alfresco-ng2-components/blob/98c0a3c7bef00e6069a279ef47a01d2732784ee0/lib/core/src/lib/form/components/widgets/core/form-field-validator.ts#L180
the validator should internally call the "convertMomentTo..."

Reply:
convertToMoment is already being handled in isValidDate() so removed from here